src/web/nextui/src/app/api/eval/route.ts

```diff
@@ -1,43 +0,0 @@
-import { NextResponse } from 'next/server';
-import { v4 as uuidv4 } from 'uuid';
-
-import store from '@/app/api/eval/shareStore';
-import { IS_RUNNING_LOCALLY } from '@/constants';
-import { writeResultsToDatabase } from '@/../../../util';
-import { runDbMigrations } from '@/../../../../migrate';
-
-import type { SharedResults } from '@/../../../types';
-
-export const dynamic = IS_RUNNING_LOCALLY ? 'auto' : 'force-dynamic';
-
-export async function POST(req: Request) {
```

Comment:
I believe this route is used by people who run `promptfoo share` pointed to a self-hosted server.

Reply:
I'll take another pass through and confirm.